src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -69,7 +70,7 @@ export class ApiKeyAuthenticator {
 
     return tryAuthenticate()
       .then((response: IResponse) => {
-        const body = response.body;
+        const body = response.body as IHash;
```

Comment:
well, since we cannot use `object` (as it's yelling about missing properties) and I would really not like to use `any` then `IHash` aka ` {[property: string]: any}` is our best shot. However I think we need a *better name* for it. 

Just thinking out loud. It's like the old JObject thing we use to have in 3.5. Also it's made from JSON responses. How about "JsonObject" or sth similar? Thoughts?

@ayende @ml054 ?